packages/clay-dropdown/src/ClayDropdownBase.js

```diff
@@ -122,6 +282,19 @@ ClayDropdownBase.STATE = {
 	items: itemsValidator.required(),
 
 	/**
+	 * The position of the dropdown (either 'up', 'down' or any of the position
+	 * constants available in `Align`).
+	 * @instance
+	 * @memberof ClayDropdownBase
+	 * @type {string|number}
+	 * @default Align.BottomLeft
+	 */
+	position_: Config.setter('setterPositionFn_')
```

Comment:
If we set this internal, since we're not setting it anywhere, we don't really need it. Remove this, setterPositionFn_ and validatePosition_ methods and replace line 105 with:

```
this.alignedPosition_ = Align.align(
 	bodyElement,
 	alignElement,
 	Align.BottomLeft
 );
```